doc/whats_new.rst

```diff
@@ -22,7 +22,7 @@ Current
 Changelog
 ~~~~~~~~~
 
-- no entries yet
+- :func:`read_raw_bids` to read electrodes.tsv only if all non-bad chs have coords; warns if not-bad ch and 'n/a' coordinate, by `Adam Li`_ (`#393 <https://github.com/mne-tools/mne-bids/pull/393>`_)
```

Comment:
These docs are meant to be user facing. I would avoid abbreviations such as chs and coords